### ISSUE TITLE
Language skill bars not visible

### ISSUE THREAD

**Issue by jgammerman** [nstrayer/datadrivencv#16]
Hi @nstrayer ,

I can reproduce your entire CV apart from one thing - the language skill bars aren't visible:
 
![image](https://user-images.githubusercontent.com/8484188/82955171-23122280-9fa6-11ea-8c46-30f1aa6d8e01.png)

However, if I click and drag on where they ought to be the language names appear, albeit faintly:

![image](https://user-images.githubusercontent.com/8484188/82955206-358c5c00-9fa6-11ea-9263-a4b38c036d86.png)

Note that this is only the case when rendering to html, not PDF which works fine. Please could you advise? I suppose that I need to change something in the CV_printing_functions.R file?

Thanks

**Comment by nstrayer** [comment]
I think this is an issue with RStudio. Have you tried looking at the output of your CV in the browser rather than the RStudio viewer? The bars don't show up in my RStudio but work fine when in the browser. This is because they use a semi-fancy CSS technique that the embedded browser in RStudio doesn't have yet. 

Let me know if that works. 

I should be clear about this in the documentation!

**Comment by jgammerman** [comment]
Yep it works fine in the browser, thanks for the prompt response!
